spanner/session.go

```diff
@@ -197,7 +197,7 @@ func (sh *sessionHandle) destroy() {
 		p.trackedSessionHandles.Remove(tracked)
 		p.mu.Unlock()
 	}
-	s.destroy(false)
+	s.destroy(false, true)
```

Comment:
Is `destroy()` only called for sessions that are in use? I would expect that it could also be called for a session that is in the list of idle sessions, and in that case it was not in use.

Reply:
sessionHandle is the wrapper which is created only for transactions(to be used) so we can assume any call for destroy using sessionHandle was in use

Reply:
@olavloite added comment for the same

Reply:
Ah, that makes it clearer, thanks!